src/main/java/fiji/plugin/trackmate/action/CloseGapsByLinearInterpolationAction.java

```diff
@@ -0,0 +1,159 @@
+package fiji.plugin.trackmate.action;
+
+import fiji.plugin.trackmate.*;
+import fiji.plugin.trackmate.gui.TrackMateGUIController;
+import fiji.plugin.trackmate.gui.TrackMateWizard;
+import ij.gui.WaitForUserDialog;
+import net.imglib2.RealPoint;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.scijava.plugin.Plugin;
+
+import javax.swing.*;
+import java.util.Iterator;
+import java.util.Set;
+
+/**
+ * This action allows to close gaps in tracks by creating new intermediate spots
+ * which are located at interpolated positions. This is useful if you want to
+ * measure signal intensity changing during time, even if the spot is not
+ * visible. Thus, trackmate is utilisable for Fluorescence Recovery after
+ * Photobleaching (FRAP) analysis.
+ *
+ * Author: Robert Haase, Scientific Computing Facility, MPI-CBG,
+ * rhaase@mpi-cbg.de
+ *
+ * Date: June 2016
+ *
+ */
+public class CloseGapsByLinearInterpolationAction extends AbstractTMAction
+{
+
+	public static final ImageIcon ICON = new ImageIcon( TrackMateWizard.class.getResource( "images/spot_icon.png" ) );
+
+	public static final String NAME = "Close gaps by introducing new spots";
+
+	public static final String KEY = "CLOSE_GAPS_BY_LINEAR_INPERPOLATION";
+
+	public static final String INFO_TEXT = "<html>" +
+			"This action closes gaps in tracks by introducing new spots. The spots positions and size are calculated using linear interpolation." +
+			"</html>";
+
+	@Override
+	public void execute( final TrackMate trackmate )
+	{
+		final Model model = trackmate.getModel();
+
+		TrackModel trackModel = model.getTrackModel();
+
+		boolean changed = true;
+
+		while ( changed )
+		{
+			changed = false;
+
+			// Got through all edges, check if the frame distance between spots
+			// is larger than 1
+			Set< DefaultWeightedEdge > edges = model.getTrackModel().edgeSet();
+			for ( DefaultWeightedEdge edge : edges )
+			{
+				Spot currentSpot = trackModel.getEdgeSource( edge );
+				Spot nextSpot = trackModel.getEdgeTarget( edge );
```

Comment:
Careful here: You assume that the target is always subsequent in time. 
This is true for "normal" TrackMate but I am always worry of weird implementation where the direction of the edge would be reversed or random. 
Could make it able to handle any weird case where time order would be reversed for source and target?